public/app/features/dashboard-scene/scene/DashboardScene.tsx

```diff
@@ -176,6 +179,7 @@ export class DashboardScene extends SceneObjectBase<DashboardSceneState> {
       body: state.body ?? DefaultGridLayoutManager.fromVizPanels(),
       links: state.links ?? [],
       ...state,
+      editPane: new DashboardEditPane({}),
```

Comment:
Shall we set this behind a feature toggle? It might help us fixing stopping bleedings if we have incidents around it.

Reply:
Then I would have to make it optional in the state, which makes some other logic a bit more messy. 

It does nothing unless it's activated (which it will never be when the feature toggle is disabled)